lib/rules/require-arrow-functions.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Requires that arrow functions are used instead of anonymous function expressions.
+ * Requires that arrow functions be used instead of anonymous function expressions in callbacks.
```

Comment:
Keep the `are`. `Requires that arrow functions are used instead of anonymous function expressions in callbacks`